e2e-tests/common/utils.resource

```diff
@@ -0,0 +1,123 @@
+*** Settings ***
+Documentation       authd test resources
+
+Resource            kvm.resource
+Library             ${CURDIR}/StringUtils.py    AS    StringUtils
+Library             Process
+
+
+*** Variables ***
+${IMAGES_DIR}    ${CURDIR}/images
+${SCRIPTS_DIR}    ${CURDIR}/scripts
+
+
+*** Keywords ***
+Log In
+    PlatformVideoInput.Match Text    Not listed    120
+    PlatformHid.Keys Combo    Return
+    BuiltIn.Sleep    2
+    PlatformHid.Type String    ubuntu
+    BuiltIn.Sleep    2
+    PlatformHid.Keys Combo    Return
+    BuiltIn.Sleep    2
+    PlatformVideoInput.Match    ${IMAGES_DIR}/circle-of-friends.png    60
+
+
+Log Out
+    Run Command    gnome-session-quit --logout --no-prompt
+
+
+Open GNOME Terminal
+    Start Application    /usr/bin/gnome-terminal
+    PlatformVideoInput.Match Text    @ubuntu:~$    15
+    PlatformHid.Keys Combo    Super_L    Up
+
+
+Open GNOME Terminal In Sudo Mode
+    Open GNOME Terminal
+    Enter Sudo Mode In GNOME Terminal
+
+
+Enter Sudo Mode In GNOME Terminal
+    PlatformHid.Type String    sudo -s
+    PlatformHid.Keys Combo    Return
+    PlatformVideoInput.Match Text    [sudo] password for ubuntu
+    PlatformHid.Type String    ubuntu
+    PlatformHid.Keys Combo    Return
+    PlatformVideoInput.Match Text    root@ubuntu:/home/ubuntu#    15
+    PlatformHid.Type String    clear
+    PlatformHid.Keys Combo    Return
+    PlatformVideoInput.Match Text    root@ubuntu:/home/ubuntu#    15
+
+
+Close GNOME Terminal
+    PlatformHid.Keys Combo    Control_L    Shift_L    W
```

Comment:
Maybe check that it's still the focused app first?
Otherwise it's a very generical "Hit Cltr+Shift+W" on app.

So maybe I'd just rely on using `Alt+F4` and rename the function "Close Focused Window"